src/backend/pagemap.h

```diff
@@ -238,6 +238,23 @@ namespace snmalloc
       return body[p >> SHIFT];
     }
 
+    /**
+     * Like get, but gives mutable access to a real pagemap entry.
```

Comment:
`get` also returns a `T&`, so should allow mutation.  I'm not sure what's different here.